community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/schema/reader/PartitionedIndexReader.java

```diff
@@ -80,6 +80,13 @@ public PrimitiveLongIterator scan()
     }
 
     @Override
+    public PrimitiveLongIterator containsString( String exactTerm )
```

Comment:
Hey guys, commit for PartitionedIndexReader already merged and we can implement this now.
Just add `return partitionedOperation( reader -> reader. containsString( exactTerm ) );`
and it should be it. Do not forget about tests as well.